### DISCUSSION TITLE
Maui Components via Nuget

### DISCUSSION THREAD

**Discussion by wallym** [dotnet/maui#2673] [General]
I have found that to do things in Xamarin Forms, I always had to go find various components in Nuget.  If I want to try and upgrade what I have to Maui, how is this going to work?  Do I have to wait on all of my components to be updated?  Is there some type of ability to use existing components compatible with XF?  Any magic in this area?

TIA,
Wally

**Comment by juansturla** [comment]
They are working on a migration tool to make some existing components move into MAUI. Like the [Migration Tool](https://docs.microsoft.com/en-us/dotnet/maui/get-started/migrate) 
If we talk about the Nuget packages, surelly each provider will have to test/update their Nuet Packages and make them MAUI-Compatible, targeting .NET6 instead of .NEt Standard
